### DISCUSSION TITLE
manambahkan 2 angka di belakang koma

### DISCUSSION THREAD

**Discussion by Dio999999** [teamalgoritma/community#173] [Q&A] [answered]
bagaimana cara untuk menambahkan 2 atau lebih angka di belakang koma.
untuk kategori data int64..

Apakah data nya perlu di ubah dulu ke Float ?

misal
1.000.000 menjadi 1.000.000,00

**Comment by onlyphantom** [comment, marked as answer]
If it's only for printing / display, you can use the following method:

```
"{:.2f}".format(100)
# returns: 100.00

"{:.3f}".format(100)
# returns: 100.000

"{:.4f}".format(100)
# returns: 100.0000
```

If you intend to perform a conversion from integer, you can use `float()`:
```
float(100)
```
